custom_metrics/javascript.js

```diff
@@ -0,0 +1,81 @@
+return JSON.stringify({
+    'num-ajax-requests': (() => {
```

Comment:
@NishuGoel should we maybe do it with SQL:
```sql
SELECT
  AVG(requests_cnt) AS avj_num_ajax_requests
FROM (
  SELECT
    page,
    COUNT() AS requests_cnt
  FROM
    `httparchive.latest.requests_desktop`
  WHERE
    JSON_QUERY(payload, "$._initiator_type") IN ('xmlhttprequest', 'fetch', 'beacon')
  GROUP BY
    page )
```

Reply:
@max-ostapenko yup, good to go with, 
the custom metric gives us the output as : `{xmlhttprequest: 0, fetch: 15, beacon: 6, total: 21}`

might need to tweak the SQL or just go with the metric. Have kept it in the code anyway in [a6d3ea9](https://github.com/HTTPArchive/legacy.httparchive.org/pull/213/commits/a6d3ea97fb9286033a63d184ebdb2558520a6f9a):

Reply:
I found out that this initiator is of [a different nature](https://developer.chrome.com/docs/devtools/network/reference/#requests), so yes - we need these custom metrics to identify AJAX requests API.